arctic/store/bson_store.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, arctic_lib):
     def initialize_library(cls, arctic_lib, hashed=True, **kwargs):
         logger.info("Trying to enable sharding...")
         try:
-            enable_sharding(arctic_lib.arctic, arctic_lib.get_name(), hashed=hashed)
+            enable_sharding(arctic_lib.arctic, arctic_lib.get_name(), hashed=hashed, key='_id')
```

Comment:
Ok, but I think I need to keep hashed to comply with the interface. I propose ignoring hashed=False with a warning in the log that it's been ignored, does that sound ok?